blocks/bricks/sequence_generators.py

```diff
@@ -257,6 +266,13 @@ def cost_matrix(self, application_call, outputs, mask=None, **kwargs):
             mask=mask, return_initial_states=True, as_dict=True,
             **dict_union(inputs, states, contexts))
 
+        # Run the language model
+        if self.language_model:
+            lm_arguments = self.language_model.apply(
+                outputs, mask=mask, as_dict=True)
```

Comment:
It is extra information which we don't have during testing time. 

I guess, I should use generated outputs here.